src/components/accordion/_accordion.scss

```diff
@@ -0,0 +1,86 @@
+@import "../../settings/all";
+@import "../../tools/all";
+@import "../../helpers/all";
+
+
+@include govuk-exports("govuk/component/accordion") {
+
+  .govuk-accordion__controls {
+    text-align: right;
+  }
+
+  // This is styled to look like a link not a button
+  .govuk-accordion__expand-all {
+    @include govuk-font($size: 16);
+    display: inline;
+    border-width: 0;
+    color: $govuk-link-colour;
+    background: transparent;
+    cursor: pointer;
+  }
+
+  // Hide the body of collapsed sections
+  .govuk-accordion__section[aria-expanded="false"] .govuk-accordion__section-body {
```

Comment:
@hannalaakso so, I've ummed and ahhed on this.

My thinking was that there are some advantages to relying on the `aria-expanded` attribute. Namely that:

* it means that if someone uses alternative javascript (but the same CSS) they won't accidentally forget to include the `aria-expanded` attribute (which would otherwise be invisible, except to those using screen readers).
* it means the javascript doesn't have to update both the `aria-expanded` attribute and an `expanded` CSS class. (This is admittedly not that big a deal)

However there may be good arguments not to have the visual implementation dependent upon the accessibility API... (personally on balance I think it's probably a net positive)

The `govuk-accordion__section--expanded` class currently serves a single purpose, which is to indicate the "initial state" of the component (which defaults to collapsed but can be override to expanded). This class is then removed by the setup javascript and swapped to the `aria-expanded` role (at https://github.com/alphagov/govuk-frontend/pull/958/files#diff-75bb29fcad7b97e804de3fd00e8252d6R114) - the rationale being that it'd be bad to have an `aria-expanded` role on the initial page HTML before the javascript loads, as at that point we can't be certain whether the javascript will load or not.

Reply:
I think it would be good not to couple visuals with `aria` attributes as they serve two separate functions. I would be in favour of adding both `govuk-accordion__section--expanded` and `aria-expanded` to open sections. 

Definitely makes sense not to have an `aria-expanded` attribute on initial page load but add it with JS 👍